### PR TITLE
Improve arm64 support.

### PR DESCRIPTION
**Pull request by tolysz**
Make library compile and run on arm64 8.2 (Nvidia Xavier arm+CUDA)
`configure` is regenerated

### CODE REVIEW

**Comment by mdouze** [comment]
Unfortunately I don't have an ARM platform for testing anymore. 
Travis builds seem broken.
@beauby could you take a look at the changes in the configure?

File: utils.cpp [hunk 1]
```diff
@@ -21,7 +21,14 @@
...
 #include <omp.h>
 
+#ifdef __SSE__
 #include <immintrin.h>
+#endif
+
+#ifdef __aarch64__
+#include  <arm_neon.h>
+#endif
+
```
Comment (beauby) [inline utils.cpp]: Could you remove all includes for simd, as it is not needed in this file?

**Comment by beauby** [comment]
The `configure` part seems legit. Would you mind rebasing @tolysz?

**Comment by mdouze** [comment]
@beauby, is this failing with the current faiss as well?

**Comment by tolysz** [comment]
thanks!

**Pull request merged**